WDAC-Policy-Wizard/app/Properties/Resources.resx

```diff
@@ -428,7 +428,7 @@ are the supported macros.</value>
     <value>Choose a Location to Save your WDAC Policy</value>
   </data>
   <data name="InvalidEKUFormat_Error" xml:space="preserve">
-    <value>This EKU is not valid or could not be converted to DER encoding. E.g. of a properly formated
+    <value> is not valid or could not be converted to DER encoding. E.g. of a properly formated
```

Comment:
I think "e.g" is a latin acronym and supposed to avoid. "Example of a properly formatted" might be better. btw, I *think* formatted has two "t"s

Reply:
Good catch!

Reply:
I would use "For example" or f.e.

Reply:
This is also known as ASN.1 dot notation, to be specific.

https://datatracker.ietf.org/doc/html/rfc2252
https://datatracker.ietf.org/doc/html/rfc1157#section-3.2.6.3